File: sqlitejava/src/main/java/com/example/sqlitejava/CarDao.java
package com.example.sqlitejava;

import android.arch.persistence.room.Dao;
import android.arch.persistence.room.Insert;
import android.arch.persistence.room.Query;

import java.util.List;

/**
 * Created by LENOVO on 15/05/2018.
 */
@Dao
public interface CarDao {
    @Insert
        void insert(Car car);

    @Query("select * from Car")

    List<Car> cars();
}

File: app/src/main/java/com/example/lenovo/sqllite/CarDao.kt
package com.example.lenovo.sqllite

import android.arch.persistence.room.Dao
import android.arch.persistence.room.Insert
import android.arch.persistence.room.Query

/**
 * Created by LENOVO on 15/05/2018.
 */
@Dao
interface CarDao {
    @Insert
    fun insert(Cr :Car)

    @Query("select * from Car")
    fun cars():List<Car>
}